medical institution/pacijent.js

```diff
@@ -0,0 +1,62 @@
+import PregledPritisak from "./pregledPritisak";
+import PregledSecer from "./PregledSecer";
+import PregledHolesterol from "./pregledHolesterol";
+import format from './dateFormatter';
+
+class Pacijent {
+    constructor(ime, prezime, jmbg, brojKartona){
+        //private fields
+        let _ime = ime;
+        let _prezime = prezime;
+        let _jmbg = jmbg;
+        let _brojKartona = brojKartona;
+        let _doktor = null;
+        //getters setters
+        this.setIme = function(ime) { _ime = ime};
+        this.getIme = function() {return _ime};
+        this.setPrezime = function(prezime){_prezime = prezime};
+        this.getPrezime = function(){return _prezime};
+        this.setJmbg = function(jmbg) {_jbmg = jmbg};
+        this.getJmbg = function() {return _jmbg};
+        this.setBrojKartona = function(brojKartona) {_brojKartona = brojKartona};
+        this.getBrojKartona = function(){return _brojKartona};
+        this.setDoktor = function(doktor){_doktor = doktor};
+        this.getDoktor = function(){return _doktor};
+        console.log("[ " + format(new Date()) + " ] Kreiran pacijent " + _ime);
+
+    }
+    izaberiDoktora(doktor){
+        console.log("[ " + format(new Date()) + " ] Pacijent " + this.getIme() + " bira doktora " + doktor.getIme() + " za svog izabranog lekara.");
+        this.setDoktor(doktor);
+        let pacijenti = doktor.getPacijenti();
+        pacijenti.push(this);
+        doktor.setPacijenti(pacijenti);
+    }
```

Comment:
ovo ne treba ovdje da se nalazi, njie pacijent odgoovoran da dodaje pacijente od doktora, ovdje ti treba samo `doktor.addPacijent(this)` , u doktor klasi imas metodu addPacijent i tamo dodajes